lms/djangoapps/course_home_api/progress/v1/serializers.py

```diff
@@ -66,9 +66,9 @@ def get_is_requestable(self, cert_data):
         return cert_data.cert_status == CertificateStatuses.requesting and cert_data.request_cert_url is not None
 
 
-class CreditCourseRequirementsSerializer(serializers.Serializer):
+class RequirementSerializer(serializers.Serializer):
```

Comment:
Maybe rename this to the `CreditRequirementSerializer`. I think it's important to still call out that this is a very specific type of requirement and in the scope of the progress tab, could mean several different things